app/models/chargeback.rb

```diff
@@ -72,6 +72,13 @@ def self.build_results_for_report_chargeback(options)
     elsif options[:tag]
       vms = Vm.find_tagged_with(:all => options[:tag], :ns => "*")
       vms &= report_user.accessible_vms if report_user && report_user.self_service?
+    elsif options[:tenant]
+      tenant = Tenant.find_by(:id => options[:tenant])
```

Comment:
Ok so I changed options[:tenant] by options[:tenant_id] and the find by 

```Ruby
    elsif options[:tenant_id]
      tenant = Tenant.where(:id => options[:tenant_id]).take
````
Thanks @lpichler && @gtanzillo